docs/quickstart-aks.md

```diff
@@ -0,0 +1,336 @@
+# Quickstart: Open Service Broker for Azure on an Azure Container Service managed cluster
+
+This quickstart walks-through using the Open Service Broker for Azure (OSBA) to
```

Comment:
walks through

Reply:
@mmacy thanks for the copy edit! Most of these are also in the minikube quickstart, so I'll also fix them in there. Thanks!!